pinot-query-planner/src/test/java/org/apache/calcite/rel/rules/PinotSortExchangeCopyRuleTest.java

```diff
@@ -92,7 +92,7 @@ public void shouldMatchLimitNoOffsetNoSort() {
 
     // Then:
     ArgumentCaptor<RelNode> sortCopyCapture = ArgumentCaptor.forClass(LogicalSort.class);
-    Mockito.verify(_call, Mockito.times(1)).transformTo(sortCopyCapture.capture(), Mockito.anyMap());
+    Mockito.verify(_call, Mockito.times(1)).transformTo(sortCopyCapture.capture());
```

Comment:
@walterddr This test started to fail once I updated the mockito dependency from 3.x to 5.x. It seems to be the only test that fails after the change. I changed the verify and now it is working, but I'm not sure about the implications.